migrations/20200328131352_initial_schema.ts

```diff
@@ -45,6 +45,8 @@ export async function up(knex: Knex): Promise<void> {
         .references('id')
         .inTable('groups')
         .index()
+
+      table.boolean('isAnon').defaultTo(0)
```

Comment:
Don't modify an existing migration, create a new one instead. Also, can we write `false` in the `defaultTo` instead of `0`?

Reply:
Extra help here: https://knexjs.org/#Migrations-CLI